src/core/dbus-execute.c

```diff
@@ -1687,6 +1732,20 @@ int bus_exec_context_set_transient_property(
 
                 return 1;
 
+        } else if (streq(name, "NUMAPolicy")) {
+                int32_t type;
+
+                r = sd_bus_message_read(message, "i", &type);
+                if (r < 0)
+                        return r;
+
+                if (!mpol_is_valid(type))
```

Comment:
thinking about this, we should accept type < 0 here too, so that this can be reset again

Reply:
Hmm, but ExecContext properties can't be set later after a transient unit was started.

Reply:
well, but we allow in unit files to say this:

```
…
[Service]
NUMAMask=1 2 3 4
NUMAMask=5 6 7 8
NUMAMask=
```

hence i think we should try to also make this work:

systemd-run -p NUMMask="1 2 3 4" -p NUMAMask="5678" -p NUMAMask="" …

And yes, this has value. For example, nspawn also takes the same --property= params, and passes them to machined, machined then adds a few more settings (but prefixes them, so that the nspawn supplied ones take precedence) and passes it on to PID 1 for scope registration. In that case it might be useful to cancel out the props machined set with the ones specified on the nspawn cmdline, and we generally can do that, and hence should also support that for the NUMMask and NUMAPolicy...